site/topics/numbers/number-bases.rst

```diff
@@ -2,11 +2,251 @@
 Number Bases
 ************
 
+* Have you ever *really* thought about counting?
```

Comment:
Honestly, remove. My initial reaction is just being insulted. I can't think of a non-disparaging intro, so no intro is good enough.